yarn.lock

```diff
@@ -9,14 +9,13 @@
   dependencies:
     "@jridgewell/trace-mapping" "^0.3.0"
 
-"@architect/functions@^4.1.1":
```

Comment:
Please revert these changes, as you didn't update any `package.json`

Reply:
These is from a totally fresh install, @MichaelDeBoey. Any changes will be because of some dependency in the various workspaces sepcified it or something else depends on it. For example, this one:

```sh
$ yarn why @architect/functions
yarn why v1.22.15
[1/4] 🤔  Why do we have the module "@architect/functions"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "@architect/functions@5.0.3"
info Reasons this module exists
   - "_project_#@remix-run#architect" depends on it
   - Hoisted from "_project_#@remix-run#architect#@architect#functions"
info Disk size without dependencies: "256KB"
info Disk size with unique dependencies: "452KB"
info Disk size with transitive dependencies: "560KB"
info Number of shared dependencies: 7
✨  Done in 0.32s.
```

Are there any that you're specifically concerned about?

Reply:
If we want to update these, they should be done in a separate PR imo.

Reply:
Sure thing - I've opened it at #2163 and reverted the changes here.

Reply:
I'll leave to you to resolve, @MichaelDeBoey